libnmstate/state.py

```diff
@@ -80,6 +81,15 @@ def interfaces(self):
         """ Indexed interfaces state """
         return self._ifaces_state
 
+    def _complement_interface_empty_ip_subtrees(self):
+        """ Complement the state interfaces with:
+          - An empty IPvX subtree.
```

Comment:
I suggest the following docstring:
`Complement the interfaces' states with empty IPv4/IPv6 subtrees.`